Runtime/DotNetE2EServerApp/DataObjects/TestUser.cs

```diff
@@ -15,7 +15,7 @@ public class TestUser : ITableData
 
         public string UserId { get; set; }
 
-        public string[] Identities { get; set; }
+        public string Identities { get; set; }

```

Comment:
type 'string' and object type Identities sounds weird.  Is this just a single identity, or a comma separated list?